### ISSUE TITLE
Error: containerd-shim-runc-v1: file does not exist: unknown

### ISSUE THREAD

**Issue by alexellis** [weaveworks/wks-quickstart-firekube#12]
System info

```
alex@nuc7:~/wks-quickstart-firekube$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
alex@nuc7:~/wks-quickstart-firekube$ docker version
Client:
 Version:           18.09.6
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        481bc77
 Built:             Sat May  4 02:35:57 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.6
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       481bc77
  Built:            Sat May  4 01:59:36 2019
  OS/Arch:          linux/amd64
  Experimental:     false
alex@nuc7:~/wks-quickstart-firekube$ ignite version
FATA[0000] This program needs to run as root.           
alex@nuc7:~/wks-quickstart-firekube$ sudo ignite version
Ignite version: version.Info{Major:"0", Minor:"6", GitVersion:"v0.6.0", GitCommit:"a8220f7b4a07ed17935fa909287286406b58106e", GitTreeState:"clean", BuildDate:"2019-08-30T22:30:57Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Firecracker version: v0.17.0
alex@nuc7:~/wks-quickstart-firekube$ 
```

The error when setting up:

```
./setup.sh --git-deploy-key  ./deploy-firekube
• Using git deploy key: ./deploy-firekube
• Found jk 0.3.0
• Found footloose 0.6.1
• Found ignite 0.6.0
• Downloading https://github.com/weaveworks/wksctl/releases/download/0.7.0/wksctl-0.7.0-linux-x86_64.tar.gz
################################################################################################################################################ 100.0%
examples/footloose
examples/footloose/README.md
examples/footloose/centos7
examples/footloose/centos7/docker
examples/footloose/centos7/docker/multimaster.yaml
examples/footloose/centos7/docker/singlemaster.yaml
examples/footloose/centos7/ignite
examples/footloose/centos7/ignite/multimaster.yaml
examples/footloose/centos7/ignite/singlemaster.yaml
examples/footloose/cluster.yaml
examples/footloose/docker-config.yaml
examples/footloose/machines-multimaster.yaml
examples/footloose/machines.yaml
examples/footloose/repo-config.yaml
examples/footloose/ubuntu1804
examples/footloose/ubuntu1804/docker
examples/footloose/ubuntu1804/docker/multimaster.yaml
examples/footloose/ubuntu1804/docker/singlemaster.yaml
examples/footloose/ubuntu1804/ignite
examples/footloose/ubuntu1804/ignite/multimaster.yaml
examples/footloose/ubuntu1804/ignite/singlemaster.yaml
examples/footloose/upload-controller-image.sh
examples/footloose/upload-image.sh
examples/footloose/user.sudoers
examples/gce
examples/gce/.gitignore
examples/gce/README.md
examples/gce/cluster.yaml
examples/gce/create-instances.sh
examples/gce/create-network.sh
examples/gce/delete-instances.sh
examples/gce/delete-network.sh
examples/gce/generate-machines-manifest.js
examples/gce/generate-machines-manifest.sh
examples/gce/repo-config.yaml
examples/vagrant
examples/vagrant/.gitignore
examples/vagrant/README.md
examples/vagrant/Vagrantfile
examples/vagrant/cluster.yaml
examples/vagrant/docker-config.yaml
examples/vagrant/machines.yaml
examples/vagrant/machines2.yaml
examples/vagrant/repo-config.yaml
wksctl
• Found wksctl 0.7.0
• Creating footloose manifest
• Creating virtual machines
INFO[0000] Creating SSH key: cluster-key ...            
INFO[0001] Pulling image: weaveworks/ignite-centos:latest ... 
INFO[0006] Creating machine: firekube-node0 ...         
FATA[0025] command "ignite run weaveworks/ignite-centos:latest --name=firekube-node0 --cpus=2 --memory=1GB --size=5GB --kernel-image=weaveworks/ignite-kernel:4.19.47 --ssh=/home/alex/wks-quickstart-firekube/cluster-key.pub --ports=2222:22 --ports=6443:6443 --ports=30443:30443 --ports=30080:30080" exited with "time=\"2019-09-14T07:41:30Z\" level=info msg=\"Docker image \\\"weaveworks/ignite-centos:latest\\\" not found locally, pulling...\"\ntime=\"2019-09-14T07:41:42Z\" level=info msg=\"Starting image import...\"\ntime=\"2019-09-14T07:41:44Z\" level=info msg=\"Imported OCI image \\\"weaveworks/ignite-centos:latest\\\" (226.8 MB) to base image with UID \\\"65cb7f0e65d1a599\\\"\"\ntime=\"2019-09-14T07:41:45Z\" level=info msg=\"Created VM with ID \\\"7a71487cbbce16e1\\\" and name \\\"firekube-node0\\\"\"\ntime=\"2019-09-14T07:41:45Z\" level=info msg=\"Pulling image \\\"weaveworks/ignite:v0.6.0\\\"...\"\ntime=\"2019-09-14T07:41:48Z\" level=fatal msg=\"failed to start container for VM \\\"7a71487cbbce16e1\\\": runtime \\\"io.containerd.runc.v1\\\" binary not installed \\\"containerd-shim-runc-v1\\\": file does not exist: unknown\"\n": exit status 1 
```

I did have ignite working on its own on this computer, it does have KVM etc.

**Comment by chanwit** [comment]
please use Ignite 0.5.2 atm.
or you better wait for 0.6.1 which will be released shortly.

**Comment by dlespiau** [comment]
Thanks for the bug report! 

This is an `ignite` bug that will be addressed in 0.6.1. For now if you use ignite 0.5.3 (you can remove ignite 0.6.0 from your path and the script will download it for you), it should work. We'll bump ignite version to 0.6.1 when released and if it solves the issue people have been seeing.

Because this is really an ignite issue. Closing the issue here.
